### ISSUE TITLE
Open special URLs in the browser

### ISSUE THREAD

**Issue by Lenne231** [myflashlab/webView-ANE#7]
I would like to open special URLs in the browser and not in the WebView. I have tried the following, but i doesn't work. It doesn't stop the loading of the page and `e.param` is the current URL, not the one that is requested.

```
private function onPageStarted(e : MyWebViewEvent) : void 
{
  if (!Helper.isAppURL(e.param))
  {
    e.preventDefault();
    e.stopPropagation();
    navigateToURL(new URLRequest(e.param));
  }
}
```

**Comment by Lenne231** [comment]
We would like to license the commercial version, because the extension seems to solve many bugs web had with StageWebView on iOS. But we definetly need an Event that fires before loading a URL with a way to cancel the loading.

**Comment by myflashlab** [comment]
@Lenne231 for opening Special URLs, an easy workaround would be to communicate with the flash side and pass in the URL. if it's one of your special URLs, you can navigateToURL. otherwise, return back to JS so it can open it in webview normally.

> e.param in the page start event is correct on Android but not on iOS
we will look into this. but tell me if you have this different addresses on online or ofline URLs? or both?

**Comment by Lenne231** [comment]
Thanks!

Unfortunately the workaround won't work for us, the websites running in the web view are too complex. 

Is there a way we could get this feature for the web view? Is it hard to implement?

We are using online URLs!

**Comment by myflashlab** [comment]
I have checked the possibility of preventing the event but unfortunately that's not as easy as I thought because we are first getting this dispatch from native part which is already extended from inner webView methods and then we just reflect back that dispatch to AS3. two threads are running side by side and preventing the flash side will happen just too late to stop the native side. it will need more time to add such a feature. besides, we try to update the extensions based on the most wanted features but I have already added this to our future to-do list. Yet, if you need this badly urgent, you can always send us a pitch from http://myappsnippet.com

**Comment by myflashlab** [comment]
by the way, we just released V2.0!
